### PR TITLE
fixes #7197 - add man page for foreman-prepare-realm

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: sbin/foreman-prepare-realm [hunk 1]
```diff
@@ -17,8 +17,8 @@ exit 1
 
 die()   { echo "$@" 1>&2; exit 1; }
 
-#[ -e /usr/bin/ipa ] || die "ipa-admintools not found."
-#[ -e /etc/ipa/default.conf ] || die "/etc/ipa/default.conf not found: please register system using ipa-client-install"
+[ -e /usr/bin/ipa ] || die "ipa-admintools not found."
+[ -e /etc/ipa/default.conf ] || die "/etc/ipa/default.conf not found: please register system using ipa-client-install"
```
Comment (stbenjam) [inline sbin/foreman-prepare-realm:21]: I can move this out to a separate issue if needed, this shouldn't have been commented out.

File: man/foreman-prepare-realm.8.asciidoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+:man manual:  Foreman Proxy Manual
+
+NAME
+----
+foreman-prepare-realm - Foreman tool for creating FreeIPA principals needed for
+the realm proxy
+
+SYNOPSIS
+--------
+foreman-prepare-realm [OPTIONS]
```
Comment (bkearney) [inline man/foreman-prepare-realm.8.asciidoc:13]: minor nit.. since there are no options, I dont know if you need this :)

**Comment by bkearney** [comment]
Looks fine to me. What I do not see is how does this get delivered? If I deliver a gem, or an rpm, how does it get built? Am I missing that?

**Comment by stbenjam** [comment]
@bkearney  There is a packaging PR as well: https://github.com/theforeman/foreman-packaging/pull/427

**Comment by bkearney** [comment]
kk.. then I am good with the text and man page as is. I dont think I have approval vudu, but +1 for me,

File: man/foreman-prepare-realm.8.asciidoc (hunk 2 above)
Comment (lzap) [inline man/foreman-prepare-realm.8.asciidoc:13]: I'd like to keep it, but can you please add an example? 

```
SYNOPSIS
--------------

foreman-prepare-realm
foreman-prepare-realm admin realm-proxy
```

I like to have the easy possiblity to copy and paste from SYNOPSIS.

File: sbin/foreman-prepare-realm (hunk 1 above)
Comment (lzap) [inline sbin/foreman-prepare-realm:21]: I believe this was already solved in https://github.com/theforeman/smart-proxy/pull/231 so please remove.

**Comment by lzap** [comment]
Okay overall good.

**Comment by domcleal** [comment]
Please don't merge this until the associated packaging PR is known to be working, thanks.

File: man/foreman-prepare-realm.8.asciidoc (hunk 2 above)
Comment (stbenjam) [inline man/foreman-prepare-realm.8.asciidoc:13]: Yup, good point, thanks

**Comment by domcleal** [comment]
Merged as 98b0dd3, thanks @stbenjam!

**Pull request closed without merging**